Packs/CortexXDR/ReleaseNotes/6_1_0.md

```diff
@@ -0,0 +1,11 @@
+
+#### Playbooks
+
+##### New: Cortex XDR Alerts Handling v2
+
+- The new playbook does not get extra incident data for each alert, and does not declare any outputs. Using this playbook retains backward-compatibility while significantly reducing the size of the workplan state in the system and improving performance.
```

Comment:
```suggestion
The new playbook does not get extra incident data for each alert, and does not declare any outputs. Using this playbook retains backward-compatibility while significantly reducing the size of the workplan state in the system and improving performance.
```
Maybe it makes sense to list what the playbook does first and only then list what it does not do.